tools/topology/topology2/platform/intel/ssp-default.conf

```diff
@@ -0,0 +1,4 @@
+# Default SSP variable definitions
+Define {
+       SSP_BLOB_VERSION		0x100
+}
```

Comment:
missing new line

Reply:
ack, mtl.conf

Reply:
no this is the default config for blob ver 1 (legacy). mtl.conf has the blob ver 1.5 def in this PR...